babel-macro/package.json

```diff
@@ -7,7 +7,7 @@
   ],
   "description": "",
   "scripts": {
-    "build": "rm -rf dist && tsc && cp -r src/test/backend/ dist/test/",
+    "build": "rm -rf dist && tsc && cp -r src/test/backend dist/test/",
```

Comment:
Don't modern `tsc`s clean up enough to do this nicely and quickly?

Reply:
But the behaviour this implements _is_ confusing Seems like we're trying to fix Microsoft/TypeScript#30602 _en passant_.  It will confuse various tools (such as anything that looks at file `mtime`s.  It discourages `tsc -w`.

I don't want to second-guess TypeScript.

Reply:
It is not an uncommon pattern (e.g. https://github.com/ReactiveX/rxjs-tslint/blob/11d64176b717c5029c212d883bfc2dfc5d14d06d/package.json#L14), I just copied it from somewhere - this is just the most paranoid version of starting from a clean slate. I run `tsc -w` without cleaning directory during development but I agree it's not the best solution.